upf/upf_pfcp_api.c

```diff
@@ -803,16 +808,140 @@ lookup_nwi (u8 * name)
   return pool_elt_at_index (gtm->nwis, p[0]);
 }
 
+static u32
+generate_teid (u32 start, u32 mask)
+{
+  return random_u32(&start) & (UINT32_MAX ^ mask);
```

Comment:
0 and ~0 (0xffffffff) are reserved TEIDs, we need to make sure we never generate those ids.